libevm/VM.h

```diff
@@ -104,6 +104,9 @@ class VM: public VMFace
 	// space for code
 	bytes m_code;
 
+	/// Space for return data.
```

Comment:
Maybe clarify that this is for the data _returned from the subcall_.
Because now we have m_output described as "return bytes" and m_returnedData described as "return data", this is confusing if you don't know what's it all about